plugin-annotation/src/test/java/com/mapbox/mapboxsdk/plugins/annotation/DraggableAnnotationControllerTest.java

```diff
@@ -265,7 +265,6 @@ public void gestureOnMoveTest() {
 
         assertTrue(moved);
         verify(annotation).setGeometry(geometry);
-        verify(annotationManager).internalUpdateSource();
```

Comment:
The reason is my PR that delays calls to `internalUpdateSource` using `postUpdateSource`, to group multiple calls in immediate succession from the client into only a single O(n) call: https://github.com/maplibre/maplibre-plugins-android/pull/12

It _should_ work to verify that `postUpdateSource` is called instead. (Tests aren't running for me locally so it's hard to say for sure.)

Reply:
Let's see...

Reply:
Yes you were right! Thanks :pray: